lightning/src/ln/functional_tests.rs

```diff
@@ -94,8 +95,8 @@ fn test_channel_resumption_fail_post_funding() {
 	assert_eq!(nodes[0].node.get_and_clear_pending_msg_events(), Vec::new());
 }
 
-#[test]
-fn test_insane_channel_opens() {
+#[xtest(feature = "_test_utils")]
```

Comment:
we could control the feature at the module level, reducing verbosity.  the tradeoff is that we'll have to generate the inventory when testing locally, but maybe that's not significant.

Reply:
so `xtest(any(test, feature = "_test_utils"))` ?

we would need to figure out semantics of `#[xtest]` without qualifiers - that would mean that it's always exposed, right?